src/components/orderDetail/OrderDescription.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import {format} from '@/utils/date';
+import S from './OrderDescription.style';
+
+type Props = {
+  id: string;
+  orderMemberName: string;
+  createdAt: string;
+  pickupReservedAt: string;
+  marketId: number;
+  marketName: string;
+  orderStatus:
+    | 'ORDERED'
+    | 'ACCEPTED'
+    | 'PICKEDUP'
+    | 'CANCELED'
+    | 'NO_SHOW'
+    | 'IN_PROGRESS'
+    | 'PICKEDUP_OR_CANCELED';
+  navigation: any;
+  marketAddress: string;
+};
+
+const OrderCustomerInfo = ({
+  id,
+  marketId,
+  marketName,
+  orderMemberName,
+  createdAt,
+  pickupReservedAt,
+  orderStatus,
+  navigation,
+  marketAddress,
+}: Props) => {
+  const getOrderStatusText = () => {
+    switch (orderStatus) {
+      case 'IN_PROGRESS':
```

Comment:
```suggestion
  const getOrderStatusText = (orderStatus: Props['orderStatus']) => {
    switch (orderStatus) {
      case 'IN_PROGRESS':
```

파라미터를 추가해서 순수함수로 컴포넌트 밖에서 선언해도 될 것 같습니다.